qubes/storage/lvm.py

```diff
@@ -241,10 +232,7 @@ def _parse_lvm_cache(lvm_output):
 
 def init_cache(log=logging.getLogger('qubes.storage.lvm')):
     cmd = _init_cache_cmd
-    if os.getuid() != 0:
-        cmd = ['sudo'] + cmd
-    environ = os.environ.copy()
-    environ['LC_ALL'] = 'C.utf8'
+    environ={'LC_ALL': 'C.UTF-8', 'PATH': '/usr/sbin:/usr/bin'}
```

Comment:
```suggestion
    environ = {**os.environ, 'LC_ALL': 'C.UTF-8'}
```

Reply:
Yes, I agree, better don't throw away the whole environment. It is called from a systemd service, so you don't need to worry about random stuff user have set in shell.